.github/workflows/update.yaml

```diff
@@ -0,0 +1,60 @@
+name: update-files
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: local checkout
+        uses: actions/checkout@v3
+ 
+      - name: checkout modules-template
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_TOKEN }}
+          repository: 'boldlink/terraform-module-template'
+          ref: '0.4.0'
+          path: .tmp
+
+      - name: copy files
+        run: |
+          cp  -R ${GITHUB_WORKSPACE}/.tmp/.github/ ./
+          cp ${GITHUB_WORKSPACE}/.tmp/Makefile \
```

Comment:
I see the update of the files, what about the workflows and their configuration files?

Reply:
@hugoalmeida264 so how we did it is instead of copying individual workflows, scripts and config files, we are copying the whole `.github` directory(in there we have workflows+scripts+config files). What do you think?